docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -31,6 +44,28 @@ To follow this guide, you need:
 - Docker
 - The docker-compose plugin
 
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Quick start for Tempo](https://killercoda.com/grafana-labs/course/tempo/quick-start).
+
+It's a fully configured environment with all the dependencies already installed.
+
+![Interactive](/media/docs/tempo/tempo-ile.svg)
```

Comment:
Hi @knylander-grafana, 
Not a problem we can get that removed. We have made it standard across the other pages. Agreed it might be to big. I can remove for now and speak to the design team to see if we can make something less aggressive but still indicate that is page is part of the greater interactive sandbox effort